### PR TITLE
Group specific channel numbering to matching backend bouquets

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.9.0
- Added: Group specfic channel numbering to matching backend bouquets
- Fixed: Fix recordings last played, playcount etc. broken since prior last PR: 248
- Added: Shorten timer type descriptions to suit UI better
- Fixed: Fixed broken playback for deleted recordings

### CODE REVIEW

**Comment by phunkyfish** [comment]
Have a bug to fix here, will let you know once it's done.

**Comment by phunkyfish** [comment]
@ksooo ready now for review.

File: pvr.vuplus/addon.xml.in [hunk 1]
```diff
@@ -175,6 +175,12 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v4.9.0
+- Added: Group specfic channel numbering to matching backend bouquets
```
Comment (ksooo) [inline pvr.vuplus/addon.xml.in]: typo: specfic -> specific

File: pvr.vuplus/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,9 @@
+v4.9.0
+- Added: Group specfic channel numbering to matching backend bouquets
```
Comment (ksooo) [inline pvr.vuplus/changelog.txt]: same typo

File: pvr.vuplus/addon.xml.in (hunk 1 above)
Comment (phunkyfish) [reply pvr.vuplus/addon.xml.in]: Silly mistake.

**Comment by phunkyfish** [comment]
Ok, should be good now.

**Review by ksooo — APPROVED** [review]

**Pull request merged**